.github/workflows/publish.yaml

```diff
@@ -15,9 +15,17 @@ jobs:
       packages: write
     strategy:
       matrix:
-        os: [linux, darwin]
+        os: [linux, freebsd]
         arch: [amd64, arm64]
     steps:
+      - name: Setup base image
+        run: |
+          if [ ${{ matrix.os }} = 'freebsd' ]; then
```

Comment:
Does this mean we build the same image twice, once per arch?

Reply:
https://github.com/rchincha/zot/actions/runs/16696661720

From OS and ARCH are specified in the stackerfile now, so the right one gets picked.
For example,
```
substitution: "OS=freebsd" was used 4 times
substitution: "ARCH=arm64" was used 4 times
```